gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/AuthenticationService.java

```diff
@@ -183,26 +188,6 @@ public Boolean isInvalidated(String jwtToken) {
         return Boolean.FALSE;
     }
 
-    /**
```

Comment:
This part is just refactoring. Please move refactoring pieces to separate PR. Then we can merge and discuss the refactoring first and then discuss the actual implementation.

Reply:
This was not done to make the apiml services better structured but to avoid code duplication. It is not a good idea to split it since the changes are closely related (except typos fixes) and depends on each other.

Reply:
I am not against it, this refactoring was overdue for some time. 
I am just asking to split these refactoring changes to separate PR so it's easier to understand even later when looking into history what's going on.

Reply:
Okay, I'll do it.